cmd/corectl/main.go

```diff
@@ -8,24 +8,30 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"path/filepath"
 	"strconv"
 	"strings"
 	"time"
 
 	"chain/core/accesstoken"
 	"chain/core/config"
+	"chain/core/fileutil"
 	"chain/core/migrate"
 	"chain/crypto/ed25519"
+	"chain/database/raft"
 	"chain/database/sql"
-	chainjson "chain/encoding/json"
 	"chain/env"
 	"chain/generated/rev"
 	"chain/log"
+	"chain/protocol/bc"
 )
 
 // config vars
 var (
-	dbURL = env.String("DATABASE_URL", "postgres:///core?sslmode=disable")
+	dbURL      = env.String("DATABASE_URL", "postgres:///core?sslmode=disable")
+	listenAddr = env.String("LISTEN", ":1999")
```

Comment:
Maybe I'm not totally up to date on the thinking here. Somehow I thought the plan was to have corectl act as a client to do config and reset operations. I'm pretty sure we already expose the necessary RPCs.

Making corectl actually do raft raises questions like, it'll have to join the cluster to do its operation, but then does it remove itself at the end? Or will the user have to do that as a separate step?